test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -294,7 +321,7 @@ class test_ucp_sockaddr : public ucp_test {
             } while (ep_count == 0);
             ASSERT_EQ(1,                  ep_count);
             EXPECT_EQ(to.ep(),            poll_eps.ep);
-            EXPECT_EQ((void *)0xdeadbeef, poll_eps.user_data);
+            EXPECT_EQ(this, poll_eps.user_data);
```

Comment:
if this is the only reason to pass test fixture ptr to entity constructor, need to remove it

Reply:
the main reason is error callback arg which also has common part in base class `ucp_test::err_handler_cb`. Since it's static, need to have pointer to class. So, need to pass it to `ucp_test_base::entity::accept`

Reply:
can we somehow void changing common test infra

Reply:
probably we can, but definitely by deeper refactoring

Reply:
seems the error callback actually needs a pointer to the entity and not a pointer to the test fixture?